src/libs/ReportUtils.js

```diff
@@ -1776,6 +1784,30 @@ function buildOptimisticReportPreview(chatReport, iouReport) {
     };
 }
 
+/**
+ * Updates a report preview action that exists for an IOU report.
+ *
+ * @param {Object} iouReport
+ * @param {Object} reportPreviewAction
+ *
+ * @returns {Object}
+ */
+function updateReportPreview(iouReport, reportPreviewAction) {
+    const message = getReportPreviewMessage(iouReport, reportPreviewAction);
+    return {
+        ...reportPreviewAction,
+        created: DateUtils.getDBTime(),
+        message: [
+            {
+                html: message,
+                text: message,
+                isEdited: false,
+                type: CONST.REPORT.MESSAGE.TYPE.COMMENT,
+            },
+        ],
```

Comment:
This will re-write existing `reportPreviewAction.message[0]`. Is it fine? Original code just replaced `html` and `text` inside.

Reply:
yes, this is fine. The original code was introduced in the previous PR where we decided to improve the code in this follow-up PR, so we are safe.